src/Features/Core/Portable/Shared/TestHooks/AsynchronousOperationListener.cs

```diff
@@ -23,9 +23,6 @@ internal partial class AsynchronousOperationListener : IAsynchronousOperationLis
         public AsynchronousOperationListener()
         {
             TrackActiveTokens = Debugger.IsAttached;
-
-            // TODO: debugging only
-            TrackActiveTokens = true;
```

Comment:
Can we break this change out into a clearly separate commit? This otherwise looks like some code you changed for debugging and this slipped in by accident.